CharacterCreation/02_Classes.txt

```diff
@@ -706,7 +706,7 @@ Class Primary Attribute: Dexterity
 Starting Wealth: $1975 + 100*Luck
 
 Passives:
-* No penalty for engaging in Grappling maneuvers.
+* The Assassin may make one additional Action when in a Grapple.
```

Comment:
Nb: We should say 'one additional Action ^per turn^' so that nobody thinks it means the Assassin gets the extra action just once for the entire grapple.